.github/install-deps.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 sudo apt update
-sudo apt install --assume-yes openssl pkg-config g++ make cmake protobuf-compiler libssl-dev libclang-dev libudev-dev git lz4
+sudo apt install --assume-yes openssl pkg-config g++ make cmake protobuf-compiler libssl-dev libclang-dev libudev-dev git lz4 yarn
```

Comment:
I think you need to setup the apt repository to install yarn (https://classic.yarnpkg.com/lang/en/docs/install/#debian-stable). Also, if the runner already have `nodejs` we can just run `npm install --global yarn` since I think most of those deps aren't needed for the `ecossytem-test`.